rest_api/api/v1.Pod.adoc

```diff
@@ -164,9 +164,9 @@ Populated by the system. Read-only. More info: http://kubernetes.io/docs/user-gu
 </div></details><div style="margin-left:13px;">        <span title="(string) This pod should be co-located (affinity) or not co-located (anti-affinity) with the pods matching the labelSelector in the specified namespaces, where co-located is defined as running on a node whose value of the label with key topologyKey matches that of any node on which any of the selected pods is running. Empty topologyKey is not allowed.">topologyKey</span>:
 </div></details></details></details><div style="margin-left:13px;">  <span title="(boolean) AutomountServiceAccountToken indicates whether a service account token should be automatically mounted.">automountServiceAccountToken</span>:
 </div><details><summary>  <span title="(array) List of containers belonging to the pod. Containers cannot currently be added or removed. There must be at least one container in a Pod. Cannot be updated.">containers</span>:
-</summary><details><summary>  - <span title="(array) Arguments to the entrypoint. The docker image&#39;s CMD is used if this is not provided. Variable references $(VAR_NAME) are expanded using the container&#39;s environment. If a variable cannot be resolved, the reference in the input string will be unchanged. The $(VAR_NAME) syntax can be escaped with a double $$, ie: $$(VAR_NAME). Escaped references will never be expanded, regardless of whether the variable exists or not. Cannot be updated. More info: https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#running-a-command-in-a-shell">args</span>:
```

Comment:
We don't manually change the API doc, so all these need to be reverted.

Reply:
Reverted